### PR TITLE
update for go-datastore

### PR DESCRIPTION
**Pull request by Stebalien**
1. gx import go-datastore
2. implement GetSize

### CODE REVIEW

**Review by magik6k — APPROVED** [review]
Oh, fun, we have a Redis datastore!

**Comment by Stebalien** [comment]
Of course!

File: redis.go [hunk 1]
```diff
@@ -60,6 +60,12 @@ func (ds *Datastore) Get(key datastore.Key) (value []byte, err error) {
 	return ds.client.Cmd("GET", key.String()).Bytes()
 }
 
+func (ds *Datastore) GetSize(key datastore.Key) (size int, err error) {
+	ds.mu.Lock()
+	defer ds.mu.Unlock()
+	return ds.client.Cmd("STRLEN", key.String()).Int()
```
Comment (schomatis) [inline redis.go:66]: From what I'm reading [`string`](https://redis.io/topics/data-types) in Redis means just arbitrary binary data so this function is safe, right?

Comment (Stebalien) [reply redis.go:66]: It should be (although I'm unfamiliar with rRdis). I just want the number of bytes in the block which should correspond to STRLEN.

**Pull request merged**